.github/autoissuelabeler.yml

```diff
@@ -79,6 +79,10 @@
   title: "(?i).*metrics.*"
   description: "(?i).*metrics.*"
   notify: [jmartisk]
+- labels: [area/metrics]
+  title: "(?i).*micrometer.*"
+  description: "(?i).*micrometer.*"
+  notify: [ebullient]
```

Comment:
And @jmartisk too?

Reply:
Given that Micrometer is mostly done @ebullient, I think that she can ping Jan when needed